assets/scripts/admin/admin_core.js

```diff
@@ -698,7 +698,7 @@ if ('ab'.substr(-1) != 'b') {
 function linksInDialog()
 {
     $(function () {
-        var iframe = $('<iframe id="dialog" allowfullscreen></iframe>');
+        var iframe = $('<iframe id="dialog" title='+$(this).attr("title")+' allowfullscreen></iframe>');
```

Comment:
I'm unsure of the dynamic iframe real accessibility :. (not RGAA or 508, but : real accessibility ;) )